src/JestConfigEditor.ts

```diff
@@ -0,0 +1,35 @@
+import { Config, ConfigEditor } from 'stryker-api/config';
+import DefaultConfigLoader from './configLoaders/DefaultConfigLoader';
+import ConfigLoader from './configLoaders/ConfigLoader';
+import * as fs from 'fs';
+import ReactConfigLoader from './configLoaders/ReactConfigLoader';
+
+export default class JestConfigEditor implements ConfigEditor {
+  public edit(config: Config): void {
```

Comment:
Perhaps it would be clearer to rename `config` to `strykerConfig`. That way you wont have `config.jest.config` and it will be clear what each config really is about.